domains/www.priyansh.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "priyanshprajapat",
+    "email": "priy@nsh.is-a.dev"
+  },
+  "record": {
+    "CNAME": "priyansh.pages.dev"
```

```suggestion
    "URL": "https://priyansh.is-a.dev"
```

You can use this but of you want you can just make it redirect.